internal/handshake/retry.go

```diff
@@ -11,11 +11,12 @@ import (
 )
 
 var (
-	retryAEADv1 cipher.AEAD // used for QUIC v1 (RFC 9000)
-	retryAEADv2 cipher.AEAD // used for QUIC v2 (RFC 9369)
+	initRetryOnce sync.Once
+	retryAEADv1   cipher.AEAD // used for QUIC v1 (RFC 9000)
+	retryAEADv2   cipher.AEAD // used for QUIC v2 (RFC 9369)
 )
 
-func init() {
+func initRetry() {
```

Comment:
Please  add a comment here, explaining why this isn't done in `init`. It's probably also a good idea to link to your issue.